Source/WebCore/animation/KeyframeAnimationOptions.h

```diff
@@ -28,14 +28,18 @@
 #include "AnimationFrameRate.h"
 #include "AnimationFrameRatePreset.h"
 #include "AnimationTimeline.h"
+#include "CSSKeywordValue.h"
 #include "KeyframeEffectOptions.h"
+#include "TimelineRangeOffset.h"
 
 namespace WebCore {
 
 struct KeyframeAnimationOptions : KeyframeEffectOptions {
     String id;
     std::optional<RefPtr<AnimationTimeline>> timeline;
     std::variant<FramesPerSecond, AnimationFrameRatePreset> frameRate;
+    std::variant<TimelineRangeOffset, RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>, String> rangeStart;
```

Comment:
`std::variant<TimelineRangeOffset, RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>, String>` is used in quite a few places. Might be worth `typedef`'ing it in `WebAnimationTypes.h` for convenience, maybe as `TimelineRangeValue` or something of that ilk?

Reply:
Fixed.